pkg/engine/ipaddresses.go

```diff
@@ -51,3 +51,50 @@ func createJumpboxPublicIPAddress() PublicIPAddressARM {
 		},
 	}
 }
+
+// CreateClusterPublicIPAddress returns public ipv4 address resource for cluster
```

Comment:
I think we need a *big* note some where stating why this is needed and when it should be removed (along with the horrible hack we had to do with `dhcp6`). So when the platform improves and removes the need for these things we remove them.